tools/test_configs/RealtekInterface.json

```diff
@@ -9,7 +9,7 @@
         },
         "connect-statement" : {
             "help" : "Disabled until EMAC updated",
-            "value" : null
+            "value" : "((RTWInterface *)net)->connect(WIFI_SSID, WIFI_PASSWORD, WIFI_SECURITY, WIFI_CHANNEL)"
```

Comment:
Hadn't you needed an alternate version of this (`net->wiFiInterface()->connect`)?

Reply:
Yes. It should be net->wiFiInterface()->connect